resources/quality/xyzprinting/xyzprinting_da_vinci_jr_pro_xplus_global_0.20_normal.inst.cfg

```diff
@@ -0,0 +1,27 @@
+[general]
+version = 4
+name = Normal Quality
+definition = xyzprinting_da_vinci_jr_pro_xplus
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = normal
+weight = -1
+global_quality = True
+
+[values]
+adhesion_type = brim
+brim_width = 10.0
+cool_fan_speed = 100
+cool_fan_speed_0 = 0
+layer_height = 0.2
+layer_height_0 = 0.35
+material_bed_temperature = 40
+material_diameter = 1.75
+material_print_temperature = 210
```

Comment:
Setting the material temperature in a global quality profile? That doesn't really make sense. This setting will _never_ be used, since material profiles always override it.

Reply:
Because in quality, we add files by different printers, and same material with different printers have different temperature.
That we set temperature in quality.

Reply:
In the material profile you can set the temperature differently for different printers or even for different nozzles for those printers. For an example, see here: https://github.com/Ultimaker/fdm_materials/blob/7a3e7e81c09c968372e79872d95b9e5cf1ccd7b0/generic_pla.xml.fdm_material#L105

This temperature is indeed overridden by the material profile at the moment, because the printing temperature is always evaluated for a certain extruder. For more information, refer to the diagram at https://github.com/Ultimaker/Cura/blob/master/docs/profiles/container_stacks.md#a-machine-instance . Settings evaluated for extruders start at the top of the extruder stack. This setting override is in the `quality` header of the global stack, which is underneath the material profiles of the exturder stacks.

Reply:
hi @Ghostkeeper  ,thank you
i have add the setting in fdm_materials(XYZprinting) help everything is going right,thanks.